packages/tools/devtools/devtools/src/index.ts

```diff
@@ -26,7 +26,7 @@ import {
 	type IFluidDevtools as IDevtoolsBase,
 	type IDevtoolsLogger,
 	initializeDevtools as initializeDevtoolsBase,
-} from "@fluidframework/devtools-core";
+} from "@fluidframework/devtools-core/internal";
```

Comment:
I'm a bit torn here... all of these things are @beta, with the intention to move to public at some point. When they are public, I think we should be importing them from the default subpath in devtools-core, right? Or are we planning to change all imports in the codebase to be from the internal subpath?

And while these things are @beta, should we be doing an import from that subpath instead of `internal`?

cc @Josmithr